server/config/config.json

```diff
@@ -13,11 +13,11 @@
     "host": "127.0.0.1",
     "dialect": "postgres"
   },
-  "production": {
-    "username": "root",
-    "password": null,
-    "database": "database_production",
-    "host": "127.0.0.1",
+   "production": {
+    "username": "snlxhrqvmtwnda",
```

Comment:
It's not ideal to push your production details to GitHub. I can hack into your production database with this info.